extensions/sonos/src/decrease-volume.tsx

```diff
@@ -29,6 +29,11 @@ export default async function Command() {
       failureMessage: `Failed to launch "Set Active Group" automatically`,
     });
   } else {
-    await coordinator.SetRelativeGroupVolume(-Number(preferences.volumeStep));
+    await coordinator.SetRelativeGroupVolume(-Number(preferences.volumeStep)).then((volume) =>
```

Comment:
Sorry to be nitpicking, bit I prefer not mixing async/await and then like this. It would be nice to stick with just the former ☺️

Reply:
sure. 🫡